File: firstcode/firstassignemt/newassignement/numberfinder.py
print("WELCOME TO THE NUMBER FINDER FRIEND")

firstNumber = 770 
while firstNumber <= 4200:
	condition_one = firstNumber % 7 

	if condition_one == 0:
		number_divisible_by_seven = firstNumber
		if not number_divisible_by_seven % 5 == 0:
			print(f"  {firstNumber:^6}    ", end=',')
	
	firstNumber = firstNumber + 1

File: firstcode/firstassignemt/myjavaassignment/NumberFinder.java
public class NumberFinder {
	public static void main(String[] args) {
		System.out.println("Welcome to the number finder");
		
	int number = 770;
			
	while(number <= 4200) {
		int conditionOne = number % 7;
		if(conditionOne == 0) {
			int conditionTwo = number;
			if(conditionTwo % 5 != 0) {
				System.out.print(conditionTwo + "  ,");
			}
		}
	   number = number + 1;		
	}

}
}